src/frontend/screens/Library/index.tsx

```diff
@@ -82,6 +82,24 @@ export default function Library(): JSX.Element {
   const { t } = useTranslation()
   const backToTopElement = useRef(null)
 
+  //Remember scroll position
+  useEffect(() => {
+    let scrollPosition = parseInt(storage?.getItem('scrollPosition') || '0')
+    const listing = document.querySelector('.listing')
+
+    if (listing) {
+      if (scrollPosition > 0) {
+        listing.scrollTo(0, scrollPosition)
+      }
+      listing.addEventListener('scroll', () => {
+        scrollPosition = listing.scrollTop
+      })
+    }
+    return () => {
+      storage.setItem('scrollPosition', scrollPosition.toString())
+    }
+  }, [])
```

Comment:
maybe it's better to use a `useRef` hook to avoid the `document.querySelector` (that is less `react-ish`), so if for some reason in the future the class changes this will still work

then we can add the variable holding that hook as a dependency in the dependencies array